docs/content/posts/2023-03-09-release-1.17.0.md

```diff
@@ -0,0 +1,485 @@
+and support access to job parameters in Python UDFs. This release focuses on
+improving PyFlink's functionality and usability, rather than introducing new
+major features. However, these enhancements are expected to improve the user
+experience and facilitate efficient data processing.
+
+
+## Daily Performance Benchmark 
+
+In Flink 1.17, daily performance monitoring has been integrated into the
+[#flink-dev-benchmarks Slack channel](https://apache-flink.slack.com/archives/C0471S0DFJ9). This feature is crucial in quickly
+identifying regressions and ensuring the quality of the code. Once a regression
+is identified through the Slack channel or the [speed
+center](http://codespeed.dak8s.net:8000), developers can refer to the guidance
+provided in the [Benchmark's
+wiki](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=115511847)
+to address the issue effectively. This feature helps the community take a
+proactive approach to ensuring system performance, resulting in a better product
+and increased user satisfaction.
+
+
+## Subtask Level Flame Graph
+
+Starting with Flink 1.17, Flame Graph provides "drill down" visualizations to
+the task level, which allows users to gain a more detailed understanding of the
+performance of their tasks. This feature is a significant improvement over
+previous versions of Flame Graph, as it empowers users to select a subtask of
+interest and see the corresponding flame graph. By doing so, users can identify
+specific areas where their tasks may be experiencing performance issues and take
+steps to address them. This can lead to significant improvements in the overall
+efficiency and effectiveness of their data processing pipelines.
+
+<center>
+<img src="/img/blog/2023-03-09-release-1.17.0/subtask-level-flame-graph.png" style="width:90%;margin:15px">
+</center>
+<!-- FIXME: Change to the actual release date -->
+
+## Generalized Delegation Token Support
+
+Previously, Flink supported Kerberos authentication and Hadoop based tokens.
+With
+[FLIP-272](https://cwiki.apache.org/confluence/display/FLINK/FLIP-272%3A+Generalized+delegation+token+support)
+being finalized, Flink’s delegation token framework is generalized to make it
+authentication protocol agnostic. This will allow contributors in the future
+to add support for non-hadoop compliant frameworks where the authentication
+protocol is not based on Kerberos. Additionally, 
+[FLIP-211](https://cwiki.apache.org/confluence/display/FLINK/FLIP-211%3A+Kerberos+delegation+token+framework)
+was implemented which improves Flink's interactions with Kerberos: It reduces
+the number of requests that are necessary to exchange delegation tokens in Flink.
+
+
+# Upgrade Notes
+
+The Flink community tries to ensure that upgrades are as seamless as can be. However, certain
+changes may require users to make adjustments to certain parts of the program
+when upgrading to version 1.17. Please refer to the release notes for a
```

Comment:
Please add link to the "release notes", this should be a deterministic URL `https://nightlies.apache.org/flink/flink-docs-release-1.17/release-notes/flink-1.17/`.